force-app/main/default/lwc/givingSummary/givingSummary.js

```diff
@@ -0,0 +1,86 @@
+import { api, LightningElement, wire } from 'lwc';
+import { getRecord } from 'lightning/uiRecordApi';
```

Comment:
```suggestion
import { getRecord, getFieldValue } from 'lightning/uiRecordApi';
```

This import will allow us to use the imported field references to access values on our records in the wired method.

Reply:
Great suggestion